apps/OpenSpace/ext/launcher/resources/qss/launcher.qss

```diff
@@ -61,6 +61,17 @@ LauncherWindow  QComboBox#config:disabled {
   color: rgb(225, 225, 225);
 }
 
+LauncherWindow  QMenu#newprofile {
+  background: rgb(50, 50, 50);
+  min-width: 8em;
+  max-width: 8em;
+  color: rgb(255, 255, 255);
+}
+
+LauncherWindow  QMenu#newprofile::item:selected {
+  background: rgb(100, 100, 100);
+}
+
```

Comment:
We're getting dangerously close to 50-shades-of-gray territory here 😅 >.> 

![image](https://github.com/user-attachments/assets/d3d0ee4d-a3d5-48bb-abea-4282b85d6551)